### PR TITLE
Separated courses content from markup

### PR DESCRIPTION
**Pull request by davidleger95**
Was going to make a small change and then got carried away... oops, take it or leave it ¯\\\_(ツ)\_/¯

- Renamed page component to `CoursesPage`.
- Removed unused imports.
- Added reusable `Course` Component (less copy/paste when changing the design).
- Separated course content from markup (easier to edit and update content).
- Open course websites in a new tab (so they don't lose their barrings on wesbos.com).

### CODE REVIEW

File: src/pages/courses.js [hunk 1]
```diff
@@ -53,7 +285,21 @@ const PriceTag = styled.span`
   }
 `;
 
-export default function AboutPage({ data, path }) {
+const Course = ({ course: { image, title, price, link, description } }) => (
+  <div className="course">
+    <img src={image.src} alt={image.alt} />
+    <H as="h3">{title}</H>
+    <PriceTag>{price}</PriceTag>
+    <a href={link.url} target="_blank" rel="noopener noreferrer">
```
Comment (davidleger95) [inline src/pages/courses.js:293]: This is the only functional change. I think it's better to open a new tab when linking to an external site, just so the user can easily come back without losing their place.

**Comment by wesbos** [comment]
thanks a ton! I think i'd like to keep this as regular HTML for the courses for now. I have this in an array in my course platform so maybe Ill look into sharing that data between the two sites. 

Will add the new tab code! do you want to change this? Or should I add it? 

I really appreciate this :)

**Comment by davidleger95** [comment]
No problem, this self-isolation is giving me too much spare time 😄 and I'll make a new PR with the new tab stuff.

Loving the new site design btw, Wes! 

I'll leave it up to you to close this in case you wanted to reference it later at all.

**Comment by wesbos** [comment]
thanks again :)

**Pull request closed without merging**